tools/swarm/src/compose.rs

```diff
@@ -789,8 +789,8 @@ mod tests {
                   P2P_ADDRESS: 0.0.0.0:1337
                   API_ADDRESS: 0.0.0.0:8080
                   GENESIS_PUBLIC_KEY: ed01203420F48A9EEB12513B8EB7DAF71979CE80A1013F5F341C10DCDA4F6AA19F97A9
-                  GENESIS_SIGNED_FILE: /tmp/genesis.signed.scale
-                  SUMERAGI_TRUSTED_PEERS: '[{"address":"irohad2:1339","public_key":"ed0120222832FD8DF02882F07C13554DBA5BAE10C07A97E4AE7C2114DC05E95C3E6E32"},{"address":"irohad1:1338","public_key":"ed0120ACD30C7213EF11C4EC1006C6039E4089FC39C9BD211F688B866BCA59C8073883"},{"address":"irohad3:1340","public_key":"ed0120FB35DF84B28FAF8BB5A24D6910EFD7D7B22101EB99BFC74C4213CB1E7215F91B"}]'
+                  GENESIS: /tmp/genesis.signed.scale
+                  TRUSTED_PEERS: '[{"address":"irohad2:1339","public_key":"ed0120222832FD8DF02882F07C13554DBA5BAE10C07A97E4AE7C2114DC05E95C3E6E32"},{"address":"irohad1:1338","public_key":"ed0120ACD30C7213EF11C4EC1006C6039E4089FC39C9BD211F688B866BCA59C8073883"},{"address":"irohad3:1340","public_key":"ed0120FB35DF84B28FAF8BB5A24D6910EFD7D7B22101EB99BFC74C4213CB1E7215F91B"}]'
```

Comment:
**This is a bug:** I don't see these changes reflected in `iroha_config`. `docker-compose.local.yml` will most certainly fail to `up`. `cargo test -p iroha_swarm` is going to fail as well (if not - it is another bug).

Overall, this renaming is fine.

We still don't run unit tests in CI, right?

Reply:
I believe I fixed it now

> We still don't run unit tests in CI, right?

I think that we wouldn't even catch this even if they did because CI pulls previous dev image. Not sure though

Reply:
There are unit tests in swarm that (should) ensure ENVs are used properly.